.github/workflows/playground.yaml

```diff
@@ -21,25 +23,13 @@ jobs:
           toolchain: nightly-2022-11-01
           override: true
           target: wasm32-unknown-unknown
+      - name: "Install wasm-pack"
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

Comment:
https://github.com/jetli/wasm-pack-action looks like it does the same but with caching (via `@actions/tool-cache`)?

Reply:
Thank you

Reply:
I saw you'd opened some issues on the `pages-action` repo... That action seems wonky. Impossible to get good behavior with branch names etc. The Wrangler one isn't much better but at least allows passing arguments directly to `wrangler`...

![Screen Shot 2022-12-26 at 2 38 26 PM](https://user-images.githubusercontent.com/1309177/209578994-28ca025d-7020-48e6-85de-f6aeb9e25872.png)

Reply:
Neat. In my case (company blog) I wanted the extra comments posted by `pages-action`, but I could imagine that being too spammy here.

Reply:
Why am I not seeing those comments...? I did want them but they weren't appearing with `pages-action`.

Reply:
Hmm. Did you provide `gitHubToken: ${{ secrets.GITHUB_TOKEN }}`? It might be conditional on that.

Reply:
Oh, definitely not. I was convinced it wasn't possible because there are issues about it like https://github.com/cloudflare/pages-action/issues/16.

Reply:
Anyway, I'll leave it for now.

Reply:
Now I’m remembering that the comment comes from the cloudflare-pages bot, not from github-actions, so the Actions configuration might not be relevant.

I have Git integration enabled in the CloudFlare dashboard, with **Automatic deployments** disabled, **Build command** set to `false`, and **Build comments on pull requests** enabled.